ting.entities.inc

```diff
@@ -316,6 +317,10 @@ class TingEntity extends DingEntity {
       drupal_alter('ting_online_url', $url, $this);
     }
 
+    if(strpos($url, '[useraccessinfomedia]') === 0){
+      drupal_alter('infomedia_url', $url, $this);
+    }
+
```

Comment:
This change seems to be irrelevant to the issue. It is a mistake that it made it into this pull request?